docs/changehistory/Roadmap.md

```diff
@@ -21,123 +21,129 @@ Volunteers for help on any or all of these projects are welcome. In particular,
+- [ ] Provide working examples and templates of agents deployed via Kubernetes.
+- [ ] Perform operations on iModels that are triggered by relevant events from iModelHub.
 
 ### iTwin Viewer
 
-- Create fully functional iTwin web viewing application platform, deployable on any cloud infrastructure. The Bentley iTwin Design Review product will be built from this application.
-- Provide necessary infrastructure to host Extensions within any iTwin Viewer-based solution.
+- [ ] Create fully functional iTwin web viewing application platform, deployable on any cloud infrastructure. The Bentley iTwin Design Review product will be built from this application.
+- [ ] Provide necessary infrastructure to host Extensions within any iTwin Viewer-based solution.
 
 ### Point Clouds
 
-- Support visualization of time-based differencing of large scale point clouds.
+- [ ] Support visualization of time-based differencing of large scale point clouds.
 
 ### Reality Data Sources
 
-- Implement a grid of linear transformations for large reality data.
+- [ ] Implement a grid of linear transformations for large reality data.
 
 ### Geolocated Photos and Panoramas
 
-- Improve server-side extraction of tags from photos and panoramas.
-- Incorporate geophoto package into an Extension loadable from any iTwin Viewer application.
+- [ ] Improve server-side extraction of tags from photos and panoramas.
+- [ ] Incorporate geophoto package into an Extension loadable from any iTwin Viewer application.
 
 ### Geolocation (general)
 
-- Provide a method to access geolocation gcs definition for i model and context data horizontal and vertical
-- Implement frontend reprojection when possible based on orbit gt implementation.
-- Implement backend of new geo coord service reprojection for complex gcs.
-- Obtain geoid separation from new geo coord service.
+- [X] Provide a method to access geolocation gcs definition for i model and context data horizontal and vertical
+- [ ] Implement frontend reprojection when possible based on orbit gt implementation.
+- [ ] Implement backend of new geo coord service reprojection for complex gcs.
+- [ ] Obtain geoid separation from new geo coord service.
 
 ### iModel Transformations
 
-- Create examples and templates illustrating usage of iModel-to-iModel transformations and synchronization.
+- [ ] Create examples and templates illustrating usage of iModel-to-iModel transformations and synchronization.
 
 ### iTwin.js based connector framework
 
-- Support for creating multi-process connectors using iTwin.js backends. One process links with source application api to read the source application files. It then communicates with another iTwin.js process via IPC to update the iModel.
+- [ ] Support for creating multi-process connectors using iTwin.js backends. One process links with source application api to read the source application files. It then communicates with another iTwin.js process via IPC to update the iModel.
```

Comment:
@swwilson-bsi do you consider this "completed" for roadmap purposes?

Reply:
Yes